### DISCUSSION TITLE
Scheduler node current status

### DISCUSSION THREAD

**Discussion by douvers** [jensrossbach/node-red-contrib-chronos#112] [Q&A] [answered]
When I enable or disable the scheduler node using a true or false input I would like to get the corresponding status output i.e. the Disabled or Enabled text that appears below the node in the editor. 

Can this be done as I would like to show the schedule status on the nodered dashboard?

Thank you.

**Comment by jensrossbach** [comment, marked as answer]
Hi,

actually you have two possibilities:

1. When you send the command for enabling or disabling to the scheduler node, you can be sure that the schedules are enabled or disabled because there are no conditions internally that would prevent this. Therefore you can rely on your commands and just connect the node sending the enable/disable commands additionally to the node that displays the status on your dashboard.
2. You can also add a Status node to your flow and [catch the status of the scheduler node](https://nodered.org/docs/user-guide/handling-errors#handling-status-changes). Then you can connect that Status node to the node that displays the status on your dashboard.

I hope this helps.

**Reply by douvers** [reply]
Thank you for the suggestions and I implemented the first option. 

I noticed when testing the status node & disabling the schedule the output is 'scheduler.status.disabledSchedule'. However when enabling the schedule the output is the next event scheduled which is perfect but it also logs 'scheduler.status.disabledSchedule' again twice. I checked this a few times and cleared the debug window every time between enabling & disabling and there was always an occurrence of the scheduler.status.disabledSchedule message. 

Thanks again.

**Reply by jensrossbach** [reply]
I found the problem with the double 'disabled' message. I will correct it and provide the fix with the next release.
